Formula/protozero.rb

```diff
@@ -0,0 +1,19 @@
+class Protozero < Formula
+  desc "Minimalistic protocol buffer decoder and encoder in C++"
+  homepage "https://github.com/mapbox/protozero"
+  url "https://github.com/mapbox/protozero/archive/v1.3.0.tar.gz"
+  sha256 "85f9238fa662ff06a1e364f1461846a9d377846274e7f98407307e31086cab2b"
+
+  def install
+    (include/"protozero").install Dir["include/protozero/*"]
```

Comment:
The upstream `Makefile` doesn't currently have an `install` target, but it would be useful for us and probably also for other package managers if they decide to distribute this library. Would you mind asking upstream for one or (even better) contribute one so that we can make use of it here?